.github/workflows/release.yaml

```diff
@@ -27,10 +27,63 @@ jobs:
       - uses: ./.github/actions/aqua
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set previous release tag for GoReleaser
+        run: |
+          export TAG=$(curl -s "https://api.github.com/repos/cybozu-go/cattage/releases/latest" | jq -r .tag_name)
+          echo "GORELEASER_PREVIOUS_TAG=${TAG}" >> $GITHUB_ENV
       - name: GoReleaser
         uses: goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b # v4.2.0
         with:
           version: latest
           args: release --clean
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  chart-release:
+    runs-on: ubuntu-22.04
+    needs: release
+    if: contains(needs.release.result, 'success')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set chart version
+        run: |
+          helm repo add cattage https://cybozu-go.github.io/cattage
+          helm repo update
+          
+          # get the release tag version
+          tag_version=${GITHUB_REF##*/v}
+          
+          # get the latest chart version
+          chart_version=$(helm search repo cattage -o json | jq -r 'sort_by(.version) | .[-1].version')
+          chart_patch_version=${chart_version##*.}
+          new_patch_version=$(($chart_patch_version+1))
+          
+          # if minor or major version changed, reset new patch version
+          local_version=$(cat charts/cattage/Chart.yaml | yq .version | sed "s/0-chart-patch-version-placeholder/$chart_patch_version/g")
+          [ "$local_version" != "$chart_version" ] && new_patch_version=0
+          
+          # replace placeholder with new version
+          sed --in-place "s/app-version-placeholder/$tag_version/g" charts/cattage/Chart.yaml
+          sed --in-place "s/0-chart-patch-version-placeholder/$new_patch_version/g" charts/cattage/Chart.yaml
+          sed --in-place "s/app-version-placeholder/$tag_version/g" charts/cattage/values.yaml
+      - name: Create release notes
+        run: |
+          tag_version=${GITHUB_REF##*/}
+          cat <<EOF > ./charts/cattage/RELEASE.md
+          Helm chart for cattage [$tag_version](https://github.com/cybozu-go/cattage/releases/tag/$tag_version)
+          
+          EOF
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Install Helm
```

Comment:
I think helm needs to be installed before line 50, since the helm command is executed in Set chart version.